azure-pipelines.yml

```diff
@@ -105,8 +105,22 @@ parameters:
     value: 1
   - name: UseELinksStub
     value: true
-
-  # Feature Toggle
+    
+  # AzureStorage
```

Comment:
These lines will need to be added to azure-pipelines.release.yml as well, so that they are injected into the deployed app.

Feel free to deploy the branch to dev if you'd like to test and see it working.

Reply:
added